frontend/src/components/ui/navbar.tsx

```diff
@@ -11,9 +12,13 @@ const Navbar = () => {
                 <Link href='/course_matching'>Course Matching</Link>
                 <Link href='/find_friends'>Find Friends</Link>
                 <Link href='/activities'>Join Activities</Link>
-                <Button asChild>
-                    <Link href="/login" className='font-sans text-[18px] font-[400px] tracking-[-.01em]'>Sign in</Link>
-                </Button>
+                {cookies().get('Token') 
```

Comment:
Should it be
```suggestion
                {cookies().get('JWT') 
```
since that's how the cookie is named in session.ts